spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -769,8 +778,15 @@ describe('The super header navigation', function () {
           expect($button).toHaveAttr('aria-label', 'Show navigation menu')
         })
 
-        it('updates the button’s visual state', function () {
-          expect($button).toHaveClass('gem-c-layout-super-navigation-header__open-button')
+        it('triggers the correct google analytics custom event', function () {
+          $button.click()
+          expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith('headerClicked', 'testingClosed', {
+            label: 'none'
+          })
+        })
+
+        xit('updates the button’s state', function () {
```

Comment:
This skipped test can also be removed as it's been updated: 

https://github.com/alphagov/govuk_publishing_components/blob/d4a897b33089817b476fec645cf11f43d2e5bc93/spec/javascripts/components/layout-super-navigation-header-spec.js#L772-L778